### PR TITLE
Instrument reference in project update custom form

### PR DESCRIPTION
**Pull request by am-e**
An autocomplete field was added to the form to refer to instruments while creating the project update.

### CODE REVIEW

**Comment by am-e** [comment]
An instrument can now be referred to while creating a project update using the custom form.
- Can be tested at https://dco.tw.rpi.edu/vivo

**Comment by mrpatrickwest** [comment]
just tested it with no instrument and no publication ... works
tested with one instrument and no publications ... works
tested with two instruments and two publications ... works

I'm satisfied. @zhonghao1014 , look at code and if needed make comments

File: src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddProjectUpdateGenerator.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 import edu.cornell.mannlib.vitro.webapp.edit.n3editing.VTwo.EditConfigurationVTwo;
 import edu.cornell.mannlib.vitro.webapp.edit.n3editing.VTwo.fields.IndividualsViaVClassOptions;
 import edu.cornell.mannlib.vitro.webapp.edit.n3editing.VTwo.fields.FieldVTwo;
-import edu.cornell.mannlib.vitro.webapp.edit.n3editing.configuration.preprocessors.AddPublicationToProjectUpdatePreprocessor;
```
Comment (mrpatrickwest) [inline src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddProjectUpdateGenerator.java:27]: so we don't need both of them? Just one of them?

File: src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/preprocessors/AddProjectUpdatePreprocessor.java [hunk 2]
```diff
@@ -21,9 +21,9 @@
 import edu.cornell.mannlib.vitro.webapp.edit.n3editing.VTwo.MultiValueEditSubmission;
 
 
-public class AddPublicationToProjectUpdatePreprocessor extends BaseEditSubmissionPreprocessorVTwo {
```
Comment (mrpatrickwest) [inline src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/preprocessors/AddProjectUpdatePreprocessor.java:24]: Ah ... I think I see. One class that handles both publications and instruments?

Comment (am-e) [inline src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/preprocessors/AddProjectUpdatePreprocessor.java:24]: Yes, I thought that would be cleaner, and it works.

**Comment by mrpatrickwest** [comment]
I say :shipit:

What say you @zednis ?

**Comment by mrpatrickwest** [comment]
:shipit:

go ahead and merge the pull request and delete the branch. I'll push it into production this weekend.

**Pull request merged**